apps/web/lib/app-providers-app-dir.tsx

```diff
@@ -107,9 +107,17 @@ const CustomI18nextProvider = (props: { children: React.ReactElement; i18n?: SSR
     window.document.dir = dir(locale);
   }, [locale]);
 
+  useLayoutEffect(() => {
+    setMounted(true);
+  }, []);
+
   const clientViewerI18n = useViewerI18n(locale);
   const i18n = clientViewerI18n.data?.i18n ?? props.i18n;
 
+  if (!mounted) {
```

Comment:
this is needed, otherwise this error gets thrown from next-i18next

```
⨯ Internal error: TypeError: Cannot read properties of null (reading 'useMemo')
@calcom/web:dev:     at useMemo (/Users/hbjorbj/Desktop/Codebases/cal.com/node_modules/react/cjs/react.development.js:1650:21)
@calcom/web:dev:     at AppWithTranslation (/Users/hbjorbj/Desktop/Codebases/cal.com/node_modules/next-i18next/dist/commonjs/appWithTranslation.js:68:35)
```